src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -15,6 +15,7 @@
 
 #include <common/ceph_time.h>
 #include <sqlite3.h>
+#include <sqlite_orm/sqlite_orm.h>
```

Comment:
Remove or replace with "sqlite_orm.h" (our wrapper header with compiler settings), otherwise the build will take loads of RAM